security/formatter.go

```diff
@@ -82,6 +84,11 @@ func ToJSON(vulns *Vulnerabilities) ([]byte, error) {
 	return json.MarshalIndent(vulns, "", "    ")
 }
 
+// ToRawJSON outputs vulnerabilities as raw JSON
```

Comment:
I think `raw` is not really understandable, here we want the JSON output on 1 line, right?